CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Please add your changelog entry under this comment in the correct category (Security, Fixed, Added, Changed, Deprecated, Removed - in this order).
 -->
 
+### Fixed
+* Properly handle LoadBalancer and InstancesV2 retrieval @marioreggiori (#120)
```

Comment:
maybe something like "Properly handle uninitialized optional features"? Feel free to find something better, but "retrieval" sounds not very fitting here